test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -681,12 +645,24 @@ public void testIncludeAllColumnsAsMetadata_mcSet() throws Exception {
   @Test
   public void testIncludeAllColumnsAsMetadataFalse_mcBlank() throws Exception {
     Map<String, String> moreEntries = new HashMap<String, String>();
-    moreEntries.put("adaptor.namespace", "Default");
-    moreEntries.put("db.modeOfOperation", "rowToText");
+    moreEntries.put("db.includeAllColumnsAsMetadata", "false");
     moreEntries.put("db.metadataColumns", "");
-    final Config config = createStandardConfig(moreEntries);
-    DatabaseAdaptor adaptor = new DatabaseAdaptor();
-    adaptor.init(TestHelper.createConfigAdaptorContext(config));
+    // Required for validation, but not specific to this test.
+    moreEntries.put("db.modeOfOperation", "rowToText");
+    moreEntries.put("db.uniqueKey", "not_id:int");
```

Comment:
It was to have *some* uniqueKey specified, otherwise the test fails with missing uniqueKey.